src/olympia/devhub/templates/devhub/addons/edit/basic.html

```diff
@@ -126,6 +128,19 @@
                     {{ select_cats(amo.MAX_CATEGORIES, form) }}
                   {% endif %}
                 {% endfor %}
+              {% elif editable %}
```

Comment:
Just for clarity I'd do a `and addon.type == amo.ADDON_STATICTHEME` so that it's clearer to read. The block above is relatively big.

Reply:
https://github.com/mozilla/addons-server/pull/6801/commits/13edb550e271340c49c4fb00fdfc946362b1a12c